packages/expo-clipboard/src/Clipboard.types.ts

```diff
@@ -78,3 +80,81 @@ export interface SetStringOptions {
    */
   inputFormat?: StringFormat;
 }
+
+// @needsAudit
+export interface ApplePasteButtonProps extends ViewProps {
+  /**
+   * A callback that is called with the result of the paste action.
+   * Inspect the `type` property to determine the type of the pasted data.
+   
+   * Can be one of `text` or `image`
+   *
+   * @example
+   * ```ts
+   *   onPress={(data) => {
+   *     if (data.type === 'image') {
+   *       setImageData(data);
+   *    } else {
+   *       setTextData(data);
+   *     }
+   *   }}
+   * ```
+   */
+  onPress: (data: PasteEventPayload) => void;
+  /**
+   * The backgroundColor of the button.
+   */
+  backgroundColor?: string;
+  /**
+   * The foregroundColor of the button.
+   */
+  foregroundColor?: string;
+  /**
+   * The cornerStyle of the button.
+   * @default capsule
+   *
+   * @see [Apple
+   * Documetation](https://developer.apple.com/documentation/uikit/uibutton/configuration/cornerstyle) for more details.
```

Comment:
```suggestion
   * @see [Apple Documentation](https://developer.apple.com/documentation/uikit/uibutton/configuration/cornerstyle) for more details.
```

Reply:
I don't think this renders correctly when it's split like this 🤔

Reply:
I think it's the way the suggested change looks here. I thought you wanted a line break betweeen `]` and `(` 😅 I'll change it back to a single line